packages/perspective-jupyterlab/src/less/index.less

```diff
@@ -15,16 +15,27 @@ div.PSPContainer {
     height: 520px;
 }
 
+// override for small style
+.jp-CodeCell.jp-mod-outputsScrolled .jp-Cell-outputArea {
```

Comment:
Undoes the breaking change here:
https://github.com/jupyterlab/jupyterlab/pull/11785